feature-libs/order/assets/translations/en/order.i18n.ts

```diff
@@ -182,4 +182,20 @@ export const order = {
     cancelSuccess: 'Your return request ({{rma}}) was cancelled',
     caption: 'Order contents.',
   },
+  reorder: {
+    button: 'Reorder',
+    dialog: {
+      reorderProducts: 'Reorder products',
+      messages: {
+        reviewConfiguration:
+          'An error occurred with "{{ productCode}}" configuration.',
+        lowStock:
+          'Quantity for {{ productName }}: {{ quantity }} has been reduced to {{ quantityAdded }}.',
+        noStock: '{{ productName }} is currently out of stock.',
+        pricingError: 'Pricing problem with "{{ productCode }}".',
+        unresolvableIssues: 'Unrecognized problem with "{{ productCode }}".',
```

Comment:
Apart from `success` I don't see where the 5 other messages are used.  Are the used?  If so where/how?